pandas/io/orc.py

```diff
@@ -87,8 +101,18 @@ def read_orc(
     )
 
     with get_handle(path, "rb", is_text=False) as handles:
-        orc_file = orc.ORCFile(handles.handle)
-        pa_table = orc_file.read(columns=columns, **kwargs)
+        source = handles.handle
+        if is_fsspec_url(path) and filesystem is None:
```

Comment:
Does this changed anything for the user (except performance)? If yes, should we mention this in the whatsnew as well?

Reply:
Don't think so, no. Looks like `get_handle` already handles URI style paths